src/modules/Dropdown/Dropdown.js

```diff
@@ -420,7 +420,11 @@ export default class Dropdown extends Component {
       if (!this.isMouseDown) {
         const { closeOnBlur } = this.props
         debug('mouse is not down, closing')
-        if (closeOnBlur) this.close()
+        if (closeOnBlur) {
```

Comment:
@levithomason this is what I meant. With this piece of code, when I tab away from this dropdown, it stays open until a user clicks somewhere else on the screen. Let me know your thoughts.